cosi.proto

```diff
@@ -97,22 +150,13 @@ message ProvisionerCreateBucketRequest {
 
```

Comment:
Expand the comment for `bucket_name` to explain purpose (e.g. `Idempotency` and `Suggested name`).